.github/workflows/main.yml

```diff
@@ -13,12 +13,17 @@ jobs:
   build-and-test:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, linux-gpu-cuda]
+        os: [ubuntu-latest, macos-14, linux-gpu-cuda]
```

Comment:
I think `macos-14` is ARM and `macos-13` is intel? If so would it be possible to have both?

Reply:
Done